src/aeromancy/wandb_tracker.py

```diff
@@ -47,6 +50,9 @@ def __init__(
         # For "https://github.com/x/y.git", we'd pull out "y" as the job name
         job_name = runtime_environment.git_repo_name
 
+        tags = self.tags or []
```

Comment:
or set()

Reply:
wait a minute: this is sometimes creating a `tags` local that could differ from self.tags!
E.g. if Tracker superclass tried to add to its self.tags, that seems like it could be a different object.

I suggest that Tracker.\_\_init__ be the only place with  "tags or set()"; remove this line 53; and use self.tags on 64.

Reply:
Good catch, thanks!